R/LearnerSurvXGBoost.R

```diff
@@ -0,0 +1,158 @@
+#' @template surv_learner
+#' @templateVar title CoxPH models based on gradient boosted trees
+#' @templateVar fullname LearnerSurvXgboost
+#' @templateVar caller [xgboost::xgboost()]
+#' @templateVar distr by [stats::predict()]
```

Comment:
Minor point, if there's a help page for [xgboost::predict.xgboost()] (or whatever it's called) then can you please change this to that

Reply:
updated